src/main/scala/no/ndla/conceptapi/model/api/Concept.scala

```diff
@@ -27,7 +27,7 @@ case class Concept(
     @(ApiModelProperty @field)(description = "When the concept was last updated") updated: Date,
     @(ApiModelProperty @field)(description = "List of people that updated this concept") updatedBy: Option[Seq[String]],
     @(ApiModelProperty @field)(description = "All available languages of the current concept") supportedLanguages: Set[String],
-    @(ApiModelProperty @field)(description = "Article id to which the concept is connected to") articleId: Option[Long],
+    @(ApiModelProperty @field)(description = "Article ids to which the concept is connected to") articleIds: Option[Seq[Long]],
```

Comment:
Mener at det var noen frontendere som ville at lister alltid skulle være tilstede (bare tomme om det ikke fantes noe data), så da foreslår jeg at denne ikke er `Option` 
```suggestion
    @(ApiModelProperty @field)(description = "Article ids to which the concept is connected to") articleIds: Seq[Long],
```